src/components/ConferenceList.js

```diff
@@ -2,35 +2,75 @@ import { connect } from 'react-redux';
 import React from 'react';
 import ConferenceCard from './ConferenceCard';
 import SearchBar from './SearchBar';
+import DropDown from './DropDown';
 import { fetchConferenceData } from '../redux/actions';
 
 export class ConferenceList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selectedValue: 'default'
+    };
+    this.handleDropDown = this.handleDropDown.bind(this);
+  }
+
   componentDidMount() {
     this.props.fetchData();
   }
 
   searchMatches = field =>
     field.toLowerCase().search(this.props.searchParam.toLowerCase()) !== -1;
 
+  handleDropDown = event => {
+    this.setState({ selectedValue: event.target.value });
+  };
+
   render() {
     const filteredList = this.props.conferences.filter(
       item =>
         this.searchMatches(item.eventName) ||
         this.searchMatches(item.eventLocation)
     );
 
-    return (
+    const conferenceList = (
       <section className=" mw5 mw7-ns center">
         <h2 className=" w-100 sans-serif pa0 f2 tl fw2 mh0 mt4 mb3">
           Upcoming Conferences
         </h2>
-        <SearchBar />
         {filteredList &&
           filteredList.map(conference => (
             <ConferenceCard key={conference.id} conferenceData={conference} />
           ))}
       </section>
     );
+
+    const filteredByDate = this.props.conferences.filter(
+      conference => new Date(conference.submissionDueDate) - new Date() < 7
+    );
+
+    const filteredConferenceList = (
+      <section>
+        <h2 className=" w-100 sans-serif pa0 f2 tl fw2 mh0 mt4 mb3">
```

Comment:
The `conferenceList` and `filteredConferenceList` components have a couple of parts that are common, the `section` (which I'm assuming should have the same classes) and the `h2`. 

I was thinking to leave those parts in the return statement, but add the conditionals like this:
```js
<section className="mw5 mw7-ns center">
   <h2 className="w-100 sans-serif pa0 f2 tl fw2 mh0 mt4 mb3">
      {isDefault ? "Upcoming Conferences" : "Events with Upcoming Submissions"}
   </h2>
   {isDefault ? conferenceList : filteredConferenceList}
</section>
```

The `conferenceList` and `filteredList` would be the array maps that you had before.